src/Chip/common/ChipSwatch.tsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+
+import { CloseIcon } from "../../icons";
+import { Chip, ChipProps } from "../Chip";
+import { ChipAdornment } from "../ChipAdornment";
+import { ColorSwatch } from "../ColorSwatch";
+
+export interface ChipSwatchProps extends Omit<ChipProps, "startAdornment"> {
+  onRemove?: () => void;
```

Comment:
What about read-only Chip w/ color swatch? If the value is omitted we shouldn't be using an empty remove button

Reply:
yes, that makes sense - made the close icon hidden when no onRemove is passed ;)